config/jobs/image-pushing/k8s-staging-test-infra.yaml

```diff
@@ -334,7 +334,8 @@ postsubmits:
           args:
           - --scratch-bucket=gs://k8s-staging-test-infra-gcb
           - --project=k8s-staging-test-infra
-          - /
+          - --build-dir=.
+          - .
```

Comment:
this will be in the repo root, which is indeed preferable to `/` which is the container filesystem room (sorry we missed this)